src/main/java/com/sap/sgs/phosphor/fosstars/data/github/GitHubDataCache.java

```diff
@@ -71,7 +73,7 @@ public int size() {
    * Returns the maximum size of the cache.
    */
   public int maxSize() {
-    return entries.maxSize();
+    return CACHE_CAPACITY;
```

Comment:
Is this update required?

Reply:
With the wrapping of the LRUMap I had to change the `entrie` field to `Map`.
As result I need to cast the `entries` to access the max size method.
Or I thought the max size is always the same as the static final value of `CACHE_CAPACITY`.

But I'm open for other options.

Reply:
Okay.